CHANGELOG.md

```diff
@@ -8,7 +8,23 @@ the following form: YYYY.0M.0D.
 
 ## [Unreleased]
 
+## [2024.09.27]
+
+### Added
+
+- Added some kernel method
+- made a java file in src where I implemented all the kernel and secondary methods.
+
+### Changed
+
+-
+
+### Fixed
+
+### Removed
+
 ## [2024.08.07]
+## [2024.10.03]
```

Comment:
Nice work! Make sure you put the newest date at the top of the file, so the latest changes are the first thing we see.